discovery-provider/src/queries/notifications.py

```diff
@@ -881,6 +889,93 @@ def notifications():
     )
 
 
+@bp.route("/solana_notifications", methods=("GET",))
+def solana_notifications():
+    """
+    Fetches the notifications events that occurred between the given slot numbers
+
+    URL Params:
+        min_slot_number: (int) The start slot number for querying for notifications
+        max_slot_number?: (int) The end slot number for querying for notifications
+
+    Response - Json object w/ the following fields
+        notifications: Array of notifications of shape:
+            type: 'ChallengeReward'
+            slot: (int) slot number of notification
+            initiator: (int) the user id that caused this notification
+            metadata?: (any) additional information about the notification
+                challenge_id?: (int) completed challenge id for challenge reward notifications
+
+        info: Dictionary of metadata w/ min_slot_number & max_slot_number fields
+    """
+    db = get_db_read_replica()
+    min_slot_number = request.args.get("min_slot_number", type=int)
+    max_slot_number = request.args.get("max_slot_number", type=int)
+
+    # Max slot number is not explicitly required (yet)
+    if not min_slot_number and min_slot_number != 0:
+        return api_helpers.error_response({"msg": "Missing min slot number"}, 500)
```

Comment:
500 is server error, probs want 400 instead - bad request

Reply:
The error response for missing min block in the other endpoint also returns a 500. Should that be updated also and is there anything that relies on or checks for the 500?